### ISSUE TITLE
Missing .podspec file

### ISSUE THREAD

**Issue by maciejimielowski** [whitedogg13/react-native-nfc-manager#76]
While doing **Install with cocopods** part of setup and running `pod install` I got following message:

> No podspec found for 'react-native-nfc-manager' in '../node_modules/react-native-nfc-manager' 

While there is `react-native-nfc-manager.podspec` file in your repository, there is no such file in node_modules/react-native-nfc-manager directory. Additionally it isn't included in files in package.json
```
  "files": [
    "NfcManager.js",
    "NdefParser.js",
    "index.d.ts",
    "android",
    "ios",
    "example"
  ],
```
Is running `pod install` not necessary?

**Comment by whitedogg13** [comment]
Sorry for the late response, it's indeed missing and I will add it back. Thanks for spotting this issue!

**Comment by jlkalberer** [comment]
Any update on this or a workaround?

**Comment by whitedogg13** [comment]
Hi, sorry for the delay. I have just added the podspec back to released npm module in latest v0.5.5 release. Thanks!
